<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>healthyselfcheck</class>
 <widget class="QWidget" name="healthyselfcheck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1215</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>健康自测问卷</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>2)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbMale">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>133</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>男</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbFemale">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>133</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>女</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbFever">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>110</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>发烧</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbCough">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>110</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>咳嗽</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbHeadache">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>110</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>头痛</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSubmit">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>90</y>
     <width>106</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>性别</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>症状</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
